inventory/host_vars/staging.katuma.org/config.yml

```diff
@@ -11,3 +11,6 @@ users_sysadmin:
   - dani
   - filipe
   - andy
+
+# Images settings
+attachment_path: ":rails_root/public/spree/products/:id/:style/:basename.:extension"
```

Comment:
the only difference here is the :rails_root, do we really need this, I dont think so...